opensrp-chw-core/src/main/java/org/smartregister/chw/core/domain/ReferralNotificationRecord.java

```diff
@@ -0,0 +1,77 @@
+package org.smartregister.chw.core.domain;
+
+public class ReferralNotificationRecord {
+
+    private String clientName;
+    private String clientDateOfBirth;
+    private String phone;
+    private String village;
+    private String notificationDate;
+    private String actionTaken;
+    private String dangerSigns;
```

Comment:
How should multiple values be handled? E.g. Multiple danger signs? 
Should these be lists instead?

Reply:
This entity is used to hold one client records at a time since it is mapping to one referral query so there is not an instance where returning multiple danger sign for an individual would occur. Danger Signs is an optional field for only ANC referrals.